### PR TITLE
Feature/1196/create table about us

### PR DESCRIPTION
**Pull request by Filiczini**
1) Create About Us page
2) Add table with mocked data to show data
3) Add test for component

#### Screenshots
image

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/about-us/tests/about-us.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  });
+  afterEach(() => {
+    jest.restoreAllMocks();
+    wrapper = null;
+  });
+
+  it('Should render AboutUs', () => {
+    expect(wrapper).toBeDefined();
+  });
+  it('Component TableContainerRow should exist', () => {
```
Comment (olga-gchak) [inline src/pages/about-us/tests/about-us.spec.js:43]: Before each "it...", please, add empy line

File: src/pages/about-us/tests/about-us.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    expect(wrapper.exists(TableContainerRow)).toBe(true);
```
Comment (olga-gchak) [inline src/pages/about-us/tests/about-us.spec.js:44]: Before each "expect...", please, add empy line

**Review by olga-gchak — CHANGES_REQUESTED** [review]

File: src/pages/about-us/about-us.js [hunk 3]
```diff
@@ -4,12 +4,33 @@ import { Link } from 'react-router-dom';
...
 const { pathToAboutUsAdd } = config.routes;
 
 const AboutUs = () => {
   const styles = useStyles();
+  const aboutUsHeaderItem = tableHeader.map(({ id, text }) => (
+    <TableContainerRow showAvatar={false} key={id} title={text} />
+  ));
+  const aboutUsItems = tableItems.map(({ id, title, text, image }) => (
+    <TableContainerRow
+      showAvatar={false}
```
Comment (olga-gchak) [inline src/pages/about-us/about-us.js:24]: Regarding this avatar, there are some data about author on the mokaps under the author's avatars, will you add it?

File: src/pages/about-us/tests/about-us.spec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      </BrowserRouter>
+    );
+  });
+  afterEach(() => {
+    jest.restoreAllMocks();
+    wrapper = null;
+  });
+
+  it('Should render AboutUs', () => {
+    expect(wrapper).toBeDefined();
```
Comment (olga-gchak) [inline src/pages/about-us/tests/about-us.spec.js:41]: What checks this test?

File: src/containers/table-container-row/table-container-row.js [hunk 5]
```diff
@@ -66,7 +67,14 @@ const TableContainerRow = ({
           </Avatar>
```
Comment (Skrekliam) [inline src/containers/table-container-row/table-container-row.js:67]: This "Avatar" always will be empty

**Review by Skrekliam — CHANGES_REQUESTED** [review]

File: src/containers/table-container-row/table-container-row.js (hunk 5 above)
Comment (Filiczini) [reply src/containers/table-container-row/table-container-row.js:67]: it's common component

File: src/pages/about-us/about-us.js [hunk 6]
```diff
@@ -4,12 +4,33 @@ import { Link } from 'react-router-dom';
...
+const { aboutUsTitles } = config.tableHeadRowTitles;
+const { aboutUsFooterTitles } = config.tableHeadRowTitles;
 const { ADD_ABOUT_US } = config.buttonTitles;
 const { pathToAboutUsAdd } = config.routes;
 
 const AboutUs = () => {
   const styles = useStyles();
+  const aboutUsHeaderItem = tableHeader.map(({ id, text }) => (
+    <TableContainerRow showAvatar={false} key={id} title={text} />
+  ));
```
[the remark this answers is not in the document]
Comment (Filiczini) [reply src/pages/about-us/about-us.js:21]: https://www.w3.org/WAI/tutorials/tables/multi-level/#:~:text=Using%20the%20headers%20attribute%2C%20all,associated%20with%20the%20data%20cell.

File: src/pages/about-us/tests/about-us.spec.js (hunk 4 above)
Comment (Filiczini) [reply src/pages/about-us/tests/about-us.spec.js:41]: the component exist

File: src/pages/about-us/tests/about-us.spec.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+import { ThemeProvider } from '@material-ui/styles';
+import { BrowserRouter } from 'react-router-dom';
+import { Button } from '@material-ui/core';
+import AboutUs from '../about-us';
+import { theme } from '../../../components/app/app-theme/app.theme';
+import TableContainerRow from '../../../containers/table-container-row';
+import TableContainerGenerator from '../../../containers/table-container-generator';
+
+jest.mock('react-redux');
+jest.mock('@apollo/client');
```
Comment (SirAndrii) [inline src/pages/about-us/tests/about-us.spec.js:12]: We should use MockProvider for apollo tests 
https://github.com/ita-social-projects/horondi_admin/pull/1204/files#diff-50ce74b9bff850805606026b6e29904ac865e546831116fc1ea0d4cff181dc19

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/table-container-row/table-container-row.js [hunk 8]
```diff
@@ -66,7 +67,14 @@ const TableContainerRow = ({
...
         </TableCell>
       )}
+
       {tableCells}
+      {imageVal && (
+        <TableCell className={classes.imageValue}>
+          <img src={imageVal} alt='horondi' />
```
[the remark this answers is not in the document]
Comment (Filiczini) [reply src/containers/table-container-row/table-container-row.js]: image

**Review by OlehProtsak — APPROVED** [review]

File: src/containers/table-container-row/table-container-row.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (Filiczini) [reply src/containers/table-container-row/table-container-row.js]: image

File: src/pages/about-us/about-us.js [hunk 9]
```diff
@@ -4,12 +4,33 @@ import { Link } from 'react-router-dom';
...
+  const aboutUsItems = tableItems.map(({ id, title, text, image }) => (
+    <TableContainerRow
+      showAvatar={false}
+      key={id}
+      title={title}
+      text={`${text.slice(0, 300)} ...`}
```
Comment (maksym-skreklia) [inline src/pages/about-us/about-us.js:27]: If we will have text for example with 150 symbols will it end with "..."?

Comment (Skrekliam) [inline src/pages/about-us/about-us.js:27]: If we will have text for example with 150 symbols will it end with "..."?

Comment (Filiczini) [reply src/pages/about-us/about-us.js:27]: yeap

File: src/pages/about-us/operations/about-us.queries.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
+import { gql } from '@apollo/client';
+
+export const getAllBlocks = gql`
```
Comment (maarekua) [inline src/pages/about-us/operations/about-us.queries.js:3]: Cant find usage of this query, as well as usage of apollo provider.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/table-container-row/table-container-row.js [hunk 11]
```diff
@@ -66,7 +66,14 @@ const TableContainerRow = ({
           </Avatar>
         </TableCell>
       )}
+
       {tableCells}
+      {image && (
+        <TableCell className={classes.imageValue}>
+          <img src={image} alt='horondi' />
```
[the remark this answers is not in the document]
Comment (Filiczini) [reply src/containers/table-container-row/table-container-row.js:73]: Yes, what is alternative ?

**Review by maarekua — APPROVED** [review]

**Review by OlehProtsak — APPROVED** [review]

**Review by Skrekliam — APPROVED** [review]

**Review by olga-gchak — APPROVED** [review]

**Pull request merged**